tensorboard/plugins/hparams/tf_hparams_backend/tf-hparams-backend.ts

```diff
@@ -56,17 +54,22 @@ export class Backend {
     );
   }
   listSessionGroups(listSessionGroupsRequest) {
-    return this._sendRequest('session_groups', listSessionGroupsRequest);
+    return this._sendRequest('/session_groups', listSessionGroupsRequest);
   }
   listMetricEvals(listMetricEvalsRequest) {
-    return this._sendRequest('metric_evals', listMetricEvalsRequest);
+    return this._sendRequest('/metric_evals', listMetricEvalsRequest);
   }
   // ---- Private methods below -------------------------------------------
   _sendRequest(methodName, request_proto) {
     if (this._useHttpGet) {
       const encodedRequest = encodeURIComponent(JSON.stringify(request_proto));
```

Comment:
You're right, thanks for catching this. Originally, the router was only called once to generate the base URL and any additional query parameters were manually added on (so they needed to do their own encoding). Since we just leverage the router throughout now, we no longer need to do the adhoc encoding.